src/pages/partners/payment-agent/_who-can-apply.tsx

```diff
@@ -142,18 +142,18 @@ const WhoCanApply = () => {
                         </StyledListHeader>
                         <StyledUl>
                             <Li>
-                                <Text>
```

Comment:
Please pass `as` and `type` appropriately according to the design.

Reply:
Realized as replace on Text only. Looks like in design